pkgs/development/libraries/libqalculate/default.nix

```diff
@@ -16,7 +16,6 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ intltool pkg-config autoreconfHook doxygen ];
   buildInputs = [ curl gettext libiconv readline ];
-  configureFlags = ["--with-readline=${readline.dev}"];
```

Comment:
You are sure we do not requires this?

Reply:
99% sure. I took it away and looked at the `configure` output during the build: it found readline. Further, when I took away that line, it revealed a compilation error due to the readline version being too old (libqalculate needs the function `rl_clear_visible_line` which is in `readline70` and later in nixpkgs). I think previously we were not actually exercising the readline support at all.